app/Providers/AppServiceProvider.php

```diff
@@ -17,6 +18,9 @@ public function boot()
       View::composer(
         'partials.components.country-select','App\Http\ViewComposers\CountrySelectViewComposer'
       );
+
+      Schema::defaultStringLength(191);
```

Comment:
Its for backward compatibility for older versions of mysql. 

https://laravel-news.com/laravel-5-4-key-too-long-error